### DISCUSSION TITLE
关于文档过于分散的问题

### DISCUSSION THREAD

**Discussion by whinc** [alibaba/formily#2879] [Q&A 2.x]
目前 formily 内部分层实现，每层的逻辑分布在独立的 pacakge 中方便理解和维护，但是文档也是按照 pacakge 划分的，导致实际使用中需查询文档时，经常要打开三四个tab页卡，在 formily、fomily/core、formily/react、formily/antd 几个文档中来回跳跃，效率低而且容易在文档间迷失，希望文档能整合成一个，按功能模块划分，相同主题下再划分框架（react/vue）和组件库(antd/element等），而非反过来按内部分层实现来划分。总而言之，目前的文档更像是侧重实现者角度组织的，而非偏向使用者角度。

以上是我的小小建议，formily 是一个非常棒的项目！

**Comment by janryWang** [comment]
建议真的挺好，不过最好能直接贡献进来，因为文档的改造工作还是非常大的，能贡献出一份力的话，对社区的帮助会非常大的

**Reply by whinc** [reply]
好的，目前刚接触 formily 不久，正在熟悉中，等有一定的熟悉度和实践经验后，非常乐意出力一起建设
